### PR TITLE
Create messages.pl.yml

### PR DESCRIPTION
**Pull request by mruminski**
Polish translation

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.pl.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+an issue lists all our needs: ''
+via Paypal: 'przez PayPal'
+
+# Howto
+Form: 'Od'
+Thanks to this form: '''
+Browser addons: 'Wtyczki dla przeglądarki'
+Mobile apps: 'Aplikacje mobilne'
+Bookmarklet: 'Bookmarklet'
+Standard Firefox Add-On: 'Standardowe rozszerzenia dla Firefox'a'یرفاکس'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.pl.yml]: Is that Persian here?

**Comment by j0k3r** [comment]
It seems that MySQL & SQLite doesn't like your file. Do you have any idea?

File: src/Wallabag/CoreBundle/Resources/translations/messages.pl.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+On our support website: 'Na naszeej stronie wsparcia technicznego'
+or: 'lub'
+'wallabag is free and opensource. You can help us:': 'wallabag jest darmowy i otwartoźródłowy. Możesz nam pomóc: :'
+'by contributing to the project:': 'przez przyłączenie się do projektu:'
+an issue lists all our needs: ''
+via Paypal: 'przez PayPal'
+
+# Howto
+Form: 'Od'
+Thanks to this form: '''
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.pl.yml]: You forget to translate that line

Comment (mruminski) [inline src/Wallabag/CoreBundle/Resources/translations/messages.pl.yml]: ok I'll fix it

File: src/Wallabag/CoreBundle/Resources/translations/messages.pl.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
+#Login
+Keep me logged in: 'Zapamiętaj mnie'
+Forgot your password?: 'Zapomniałeś hasła'
+Login: 'Loguj'
+Back to login: 'Wróć do logowania'
+Send: 'Wyślij'
+'Enter your email address below and we'll send you password reset instructions.': 'Wpisz swój adres email poniżej. Wyślemy Ci instrukcję resetowania hasła'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.pl.yml]: Could you wrap the english sentence using `"` instead of `'` ?

**Comment by nicosomb** [comment]
Thank you!

**Pull request merged**